pkg/version/version.go

```diff
@@ -0,0 +1,6 @@
+package version
+
+var (
+	// Version is passed with -X ldflag, but default to 2.0.0 as it's operator v2.
+	Version = "2.0.0"
```

Comment:
What would also be beneficial is to add Commit hash so we know against which commit given build was performed.

Reply:
> What would also be beneficial is to add Commit hash so we know against which commit given build was performed.

Probably will not work with docker build since it copies just source. Any suggestion? It can be passed as a build-arg but then it's not really hiding the machinery in the Makefile. Changing CI Dockerfiles is another (least) problem.